gum/backend-dbghelp/gumsymbolutil-dbghelp.c

```diff
@@ -148,6 +152,36 @@ gum_find_functions_matching (const gchar * str)
   return matches;
 }
 
+gboolean
+gum_load_symbols (const gchar * path)
+{
+  GumDbghelpImpl * dbghelp;
+  DWORD64 base;
+  WCHAR * path_utf16;
+  gboolean success;
```

Comment:
The variable holding the result should go first as per the [style guide](https://github.com/frida/frida/blob/master/CONTRIBUTING.md).